internal/cmd/commands/credentiallibrariescmd/funcs.go

```diff
@@ -140,6 +140,16 @@ func printItemTable(item *credentiallibraries.CredentialLibrary, resp *api.Respo
 		nonAttributeMap["Type"] = item.Type
 	}
 
+	var keySubstMap map[string]string
+	switch item.Type {
+	case "vault":
+		fallthrough
+	case "vault-generic":
```

Comment:
What do you think about just
```suggestion
	case "vault", "vault-generic":
```